### DISCUSSION TITLE
Web version

### DISCUSSION THREAD

**Discussion by nvohra0016** [GLVis/glvis#207] [Q&A] [answered]
I am unable to use the web version of glvis to load my solution (.gf). Is there a particular way to do that? When I click on the "Load local file" button and select the file, nothing happens. 

Thank you!

**Comment by tzanio** [comment, marked as answer]
Hi @nvohra0016,

The online version does not support separate mesh and grid functions yet, instead it reads GLVis streams, see https://glvis.org/options-and-use/#server-mode, which usually have the extension `.saved`.

If you have a mesh `my.mesh` and grid function `my.gf`  it is easy to create a stream by concatenating them and adding `solution` on the first line as follows:

```
echo "solution" > my.saved
cat my.mesh >> my.saved
cat my.gf >> my.saved
```

See for example https://github.com/GLVis/data/blob/master/streams/quad.saved.

Alternatively you can just save the stream from the desktop GLVis application, if you run it in server mode with `glvis -mac`.

The streams are preferable, not only because they combine the mesh and grid function, but also because you can add other GLVis commands at the end, e.g. `keys`, `valuerange`, etc. see https://github.com/GLVis/data/blob/master/streams/snake.saved

Hope this helps,
Tzanio

**Reply by v-dobrev** [reply]
There is a typo above: `cat my.gf` should be `cat my.gf >> my.saved`

**Reply by tzanio** [reply]
Fixed

**Comment by nvohra0016** [comment]
Thank you very much! I'll try doing this.
